src/use/helpers/Toasts.ts

```diff
@@ -0,0 +1,32 @@
+import { useToast } from "@/toast";
+import ReportBug from "@/components/common/ReportBug.vue";
+import { markRaw } from "vue";
+
+const toast = useToast();
+
+export function showNetworkErrorToast() {
+    toast.error("Network error: make sure to establish a stable internet connection.");
+}
+
+export function showUpdateSuccessToast(subject: string) {
+    toast.success("'" + subject + "' updated.");
+}
+
+export function showUpdateFailedToast(subject: string) {
+    toast.error("Error: '" + subject + "' not updated.");
+}
+
+export function showAPIToast(code: string, msg?: string) {
```

Comment:
why is code a string, when it is only ever used for numbers?

Reply:
I changed it to number.